tuplex/utils/include/Utils.h

```diff
@@ -73,6 +73,8 @@ constexpr const char* base_file_name(const char* path) {
     return file;
 }
 
+constexpr double D_NAN = nan("");
```

Comment:
add comment: is this qnan? snan? what kind of value is that? Is the constant used in the isnan function at all?